### DISCUSSION TITLE
We found the patch versions not appeared in Golang Index.

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [zitadel/zitadel#6235] [Ideas]
Hello, we are a research team working on Golang. During our investigation, we discovered that the CVE-2023-22492 was addressed and the patch versions were released. However, we noticed that these patch versions have not appeared in the Golang Index, which means that 'go list' cannot automatically push the patch versions to downstream users.

We recommend that after releasing the versions, you push them to the Golang Index using the command 'go get github.com/zitadel/zitadel@version'. This will enable the automatic distribution of the patch versions to downstream users. Thank you for your attention.

**Comment by fforootd** [comment]
Hi

Thanks for raising this.

ZITADEL itself is only a product that is not intended to be used as golang dependency.
I think you can safely ignore this problem. In the meantime we will investigate why the package was published in the first place (could be because we published the CVE).
